tabbycat/privateurls/views.py

```diff
@@ -66,7 +66,7 @@ def build_url(person):
 
         def build_link(person):
             if person.url_key is None:
-                return {'text': _("no key"), 'class': 'text-warning'}
+                return ''
```

Comment:
Sadly emojis can't be superimposed; I was thinking an "X" over the link. Guess I forgot the text was there.

Reply:
Oh, my preference would still be no text 🙂